components.html

```diff
@@ -0,0 +1,707 @@
+          <rect width="100%" height="100%" fill="#868e96"></rect><text x="50%" class="res-text" y="50%" fill="#dee2e6"
+            dy=".3em">Responsive image</text>
+        </svg>
+        <h2>How To Create Responsive Images</h2>
+        <h3>Step 1) Add HTML:</h3>
+        <iframe src="https://carbon.now.sh/embed/8TU15ZV6YFvYGzfWOBer" class="iframe-style"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <h3>Step 2) Add CSS:</h3>
+        <p>If you want the image to scale both up and down on responsiveness, set the CSS max-width
+          property to
+          100% and height to auto:</p>
+        <iframe src="https://carbon.now.sh/embed/uXOd9ieD284EEl2luU8w" class="iframe-style"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <hr>
+        <h2>Round Images:</h2>
+        <p>Round images are very simple to make just put the "border-radius : 50% ". You need to adjust
+          the
+          width and height according to your case.</p>
+        <img class="round-img" src="https://fimapay.com/en-uk/images/w3/avatar2.png" alt="">
+        <h3>How to Create a Round Image :</h3>
+        <h3>Step 1) Add HTML :</h3>
+        <iframe src="https://carbon.now.sh/embed/MEsWqaHRNh1dCm0zhjv8" class="iframe-style"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <h3>Step 2) Add Css :</h3>
+        <iframe src="https://carbon.now.sh/embed/sH2HURt6LNyUNwKPIPRx" class="iframe-style"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+      </div>
+    </div>
+
+    <!-- Badge -->
+    <br>
+    <div id="badge-div" class="badge-contents">
+      <h1>Badges</h1>
+
+      <div class="bd-example">
+        <h1>Example heading <span class="badge badge-secondary">New</span></h1>
+        <h2>Example heading <span class="badge badge-secondary">New</span></h2>
+        <h3>Example heading <span class="badge badge-secondary">New</span></h3>
+        <h4>Example heading <span class="badge badge-secondary">New</span></h4>
+        <h5>Example heading <span class="badge badge-secondary">New</span></h5>
+        <h6>Example heading <span class="badge badge-secondary">New</span></h6>
+        <iframe src="https://carbon.now.sh/embed/lYgmyJaul6GVjuS5vdXJ" class="iframe-style"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <hr>
+        <h3>Icon Badges:</h3>
+        <div class="badge-contentdiv">
+          <div class="badge-item">
+            <div class="badge-content">
+              <i class="badgeicon fa fa-bell"></i>
+              <span class="value">5</span>
+            </div>
+          </div>
+          <div class="badge-item">
+            <div class="badge-content">
+              <i class="badgeicon fas fa-shopping-cart"></i>
+              <span class="value">5</span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- <h3>HTML:</h3>
```

Comment:
HTML and CSS comments are still counted towards the size of the files that they are in. The bigger the file, the longer it will take to download. It's good practice to remove any unused code and minimize your css/js files when you're deploying to production.